src/control/lib/telemetry/promexp/collector.go

```diff
@@ -371,6 +369,9 @@ func (c *Collector) Collect(ch chan<- prometheus.Metric) {
 			gauges.add(baseName, desc, rm.m.FloatValue(), labels)
 		case telemetry.MetricTypeDuration:
 			gauges.add(baseName, desc, rm.m.FloatValue(), labels)
+			for _, ms := range getMetricStats(baseName, desc, rm.m) {
```

Comment:
"rm.m" wasn't immediately obvious and could probably be named more clearly

Reply:
Fixed in #6200